src/blocks/opentable/preview.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * WordPress dependencies
+ */
+
+const {__} = wp.i18n;
+
+const {RichText} = wp.blockEditor;
+
+const OpenTablePreview = function( props ) {
+	const {
+		attributes: {
+			dateLabel,
+			partySizeLabel,
+			timeLabel,
+			submitButtonText,
+		},
+		setAttributes,
+	} = props;
+
+	let currentTime = new Date(),
```

Comment:
create a `getCurrentDate( separator )` function in `utils.js` to reduce duplicated code

Reply:
I have added another param that will help me to change date format, if there is the case.